### DISCUSSION TITLE
Can you add functionality to render extra jsx in modal

### DISCUSSION THREAD

**Discussion by ghost** [henninghall/react-native-date-picker#441] [Ideas]
I ended up a requirement where I need to render extra jsx into footer or header in modal view.  And also when we need to render toast in modal view on more visibility. Thanks

**Comment by henninghall** [comment]
Hi, you would have to create your own modal and use the inline picker in that case!

**Comment by ghost** [comment]
I had tried that, It caused crash problem in app when ever I tried to scroll the picker dates or months.

      <DatePicker
date={new Date(currentDate)}
mode="datetime"
androidVariant="nativeAndroid"
minimumDate={new Date(currentDate)}
maximumDate={new Date(moment(currentDate).add(23, 'd').format('YYYY-MM-DD'))}
fadeToColor={Color.WHITE_COLOR}
      modal
      open={showDateModal}
      onConfirm={(date) => {
        if (moment(date).format("d") !== '1'){
          ToastError({ message: "Only monday are available to start diet plan" })
        }
        else {
          Toast({ message: "Success! Monday choosed" })
          this.setState({
            showDateModal: false,
            selectedDate: moment(date).format('YYYY-MM-DD')
          })
        }

      }}
      onCancel={() => {
        this.setState({ showDateModal: false })
      }}

Smartphone (redmi 10):
OS: Android 11
React Native version "0.64.1",
react-native-date-picker version ^4.1.3

**Reply by henninghall** [reply]
That happens because react native version 0.64.1 is not supported, you will need to bump to 0.64.2 or later.
